compiler-plugin-test/src/test/resources/test-src/sql_script/package_19/modules/foo/entities.bal

```diff
@@ -0,0 +1,10 @@
+import ballerina/persist;
```

Comment:
License header?

Reply:
Added in [2359de4](https://github.com/ballerina-platform/module-ballerina-persist/pull/50/commits/2359de46f1ea8c84cc7c0c481c91b059b75cae8f)